### PR TITLE
Changes in configuration template and tests import

### PR DESCRIPTION
**Pull request by plomakin**
Added lost parameters for networks, vms, images.
Fixed imports for tests from load_generator.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/56/

**Comment by miarmak** [comment]
+1

File: devlab/config.template [hunk 1]
```diff
@@ -20,6 +20,9 @@ ssh_chunk_size = 100
 retry = 5
 migrate_extnets = True
 time_wait = 5
+all_vms = False
```
Comment (roman-verchikov) [inline devlab/config.template]: Don't we want this to be set to `True` by default?

Comment (plomakin) [inline devlab/config.template]: I believe it is False in CF direcory: configs/config.ini
So, i've just copied it to template as is.

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**